mne/viz/_brain/_brain.py

```diff
@@ -1163,9 +1163,15 @@ def _set_label_mode(mode):
         self.tool_bar.addWidget(QLabel("Label extraction mode"))
         self._label_mode_widget = QComboBox()
         self.tool_bar.addWidget(self._label_mode_widget)
+        if self._data["stc"] is not None:
+            vec = self._data["stc"]._data_ndim == 3
+        else:
+            vec = False
         for source in ["stc", "src"]:
             if self._data[source] is not None:
                 for mode in self.default_label_extract_modes[source]:
+                    if vec and mode == "pca_flip":
```

Comment:
It's not just `pca_flip`, `mean_flip` also will not work. Instead of duplicating checks you should DRY with this somehow:

https://github.com/mne-tools/mne-python/blob/master/mne/source_estimate.py#L3085-L3092